src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,31 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static IDataTransferManagerInterop dataTransferManagerInterop = DataTransferManager.As<IDataTransferManagerInterop>();
+
+        public static DataTransferManager GetForWindow(IntPtr appWindow)
+        {
+            Guid iid = GuidGenerator.CreateIID(typeof(IDataTransferManager));
+            return (DataTransferManager)dataTransferManagerInterop.GetForWindow(appWindow, iid);
+        }
+
+        public static void ShowShareUIForWindow(IntPtr appWindow)

```

Comment:
Should this use `WindowId` instead? Also has any of this gone through API review?

Reply:
WindowId requires 20348+ SDK or WASDK, so I think keeping it in IntPtr is okay.

Reply:
IntPtr is also what we use for other interop interfaces here.